### PR TITLE
feat(web): add a config item to web，let the developer can select whether enable the html cache

### PR DESCRIPTION
**Pull request by Tiny-Fendy**
This is a new PR related to [https://github.com/verdaccio/verdaccio/pull/3108](https://github.com/verdaccio/verdaccio/pull/3108).

### CODE REVIEW

**Comment by Tiny-Fendy** [comment]
The website has been archived, I can't add a description for this change in the docs.

**Comment by juanpicado** [comment]
> The website has been archived, I can't update the documentation for this change.

You can change the docs file directly on this https://github.com/verdaccio/verdaccio/pull/3108

It´s a bit confusing, the master branch refers to next major (v6), but also deploys the website (which refers to v5) eventually both will be alinged when v6 is released, but at this point is like this.

File: src/api/web/html/renderHTML.ts [hunk 1]
```diff
@@ -98,7 +99,9 @@ export default function renderHTML(config, manifest, manifestFiles, req, res) {
         manifest
       );
       debug('template :: %o', webPage);
-      cache.set('template', webPage);
+      if (needHtmlCache) {
+        cache.set('template', webPage);
+      }
       debug('set template cache');
```
Comment (juanpicado) [inline src/api/web/html/renderHTML.ts:105]: We should include debug `debug('set template cache');` within the if condition otherwise is missleading.

Comment (Tiny-Fendy) [reply src/api/web/html/renderHTML.ts]: OK，I have committed it.

**Review by juanpicado — APPROVED** [review]
LGTM great stuff @Tiny-Fendy it will be on next minor

File: src/api/web/html/renderHTML.ts (hunk 1 above)
Comment (juanpicado) [reply src/api/web/html/renderHTML.ts]: I've took the liberty to apply some small changes, add suggestion to the config files and just moved your new line to the header of the function close to the other configs.

**Comment by juanpicado** [comment]
Failed checks are unrelated with this changes.

**Pull request merged**